OKTA-555386 (#168)
okta/workflows-templates: workflows/execute_onpremise_powershell_with_okta_workflows/readme.md

> 1.  <span class="c0">The configuration and screenshots captured were

Reviewer: ```suggestion
1.  <span class="c0">The configuration and screenshots are
```

<span class="c5 c16 c25"></span>

> 5.  <span class="c8">Link the account ID and workspace ID with the following command:</span>

Reviewer: ```suggestion
1.  Enter this command to link the account ID and workspace ID:
```
